core/sql/sqlcomp/PrivMgrPrivileges.cpp

```diff
@@ -5390,7 +5390,8 @@ PrivStatus ObjectPrivsMDTable::updateWhere(const std::string & setClause,
 //    <catalogName> "<schema_name>"."<object_name>", 
 //    object_type,
 //    object_owner, -- granteeID
-//    (select auth_db_name from AUTHS where auth_id = object_owner), --granteeName
+//    coalesce((select auth_db_name from AUTHS where auth_id = object_owner),
+//             'DB__ROOT') --granteeName
```

Comment:
This is probably okay.  I am just wondering what happens if you initialize authorization, create tables with role and user owners other than DB__ROOT, then drop authorization.  Not sure if we reset object ownership.  Later when we initialize authorization, we may be missing things.   Some testing is needed and perhaps a new JIRA created.  Unfortunately I don't have a working version so I can't test.  I will test is out when I do and create a JIRA, if needed.